src/dapps/saga.ts

```diff
@@ -72,6 +72,13 @@ export function* handleFetchDappsList() {
   }
 
   const address = yield select(walletAddressSelector)
+  if (!address) {
+    // the dapplist is fetched on app start, but for a new user who has not yet
+    // created or restored a wallet the request will fail.
+    Logger.debug(TAG, 'Wallet address not found')
```

Comment:
🙈 i think that the code already tells us that? i wanted to explain why the wallet address could be empty, since it might not be obvious whether this is just defensive programming or a legit use case

Reply:
Sorry, I meant the logger message.